### PR TITLE
[netcore][05.multi-turn-prompt] Fix typos and improvements

### PR DESCRIPTION
**Pull request by VictorGrycuk**
## Proposed Changes
Fix several typos in sample # 05 `multi-turn-prompt`
## Testing
Sample documentation and code should not have typos

### CODE REVIEW

**Review by joseques — APPROVED** [review]

File: samples/csharp_dotnetcore/05.multi-turn-prompt/readme.md [hunk 1]
```diff
@@ -31,9 +31,7 @@ developers to test and debug their bots on localhost or running remotely through
 - File -> Open bot and navigate to `botbuilder-samples/samples/csharp_dotnetcore/05.multi-turn-prompt` folder.
 - Select `BotConfiguration.bot` file.
 # Deploy this bot to Azure
-You can use the [MSBot](https://github.com/microsoft/botbuilder-tools) Bot Builder CLI tool to clone and configure any services this sample depends on. 
-
-To install all Bot Builder tools -
```
Comment (dfavretto) [inline samples/csharp_dotnetcore/05.multi-turn-prompt/readme.md:36]: This line shouldn't be combined with the one above.

**Review by dfavretto — CHANGES_REQUESTED** [review]

File: samples/csharp_dotnetcore/05.multi-turn-prompt/readme.md (hunk 1 above)
Comment (VictorGrycuk) [reply samples/csharp_dotnetcore/05.multi-turn-prompt/readme.md:36]: The line has been fixed with the correct text

**Review by dfavretto — APPROVED** [review]
LGTM

**Comment by VictorGrycuk** [comment]
Replaced with https://github.com/Microsoft/BotBuilder-Samples/pull/620

**Pull request closed without merging**